code/part-two/processor/test/00-MojiHandler.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const expect = require('chai').expect;
+const { InvalidTransaction } = require('sawtooth-sdk/processor/exceptions');
+
+const MojiHandler = require('../handler').MojiHandler;
+const Txn = require('./mocks/txn');
+const Context = require('./mocks/context');
+
+
+describe('Core MojiHandler Behavior', function() {
+  let handler = null;
+  let context = null;
+
+  before(function() {
+    handler = new MojiHandler();
+  });
+
+  beforeEach(function() {
+    context = new Context();
+  });
+
+  it('should return a Promise', function() {
+    const txn = new Txn({ hello: 'world' });
+    expect(handler.apply(txn, context)).to.be.an.instanceOf(Promise);
+  });
+
+  it('should reject a poorly encoded payload', function() {
+    const txn = new Txn({});
+    txn.payload = Buffer.from({ hello: 'world' }.toString());
+
+    return handler.apply(txn, context)
+      .catch(err => {
+        expect(err, 'Error should be an InvalidTransaction')
+          .to.be.instanceOf(InvalidTransaction);
+        return true;
+      })
+      .then(wasRejected => {
+        expect(wasRejected, 'Transaction was not rejected').to.be.true;
```

Comment:
@delventhalz Thoughts on having all of the custom error messages for `expect` be "should" statements (like on line 34)?

eg, `Transaction was not rejected` -> `Transaction should be rejected`

If you like this idea, I saw some opportunities to apply it on a few lines in `01-CreateCollection.js` as well! Otherwise everything else lgtm.